packages/traceability-vocab/src/generate-spec.test.js

```diff
@@ -93,9 +93,21 @@ it('should validate using json schema', async () => {
       // only if everything validated with no errors should this add to the OpenAPI spec
       try {
         const $classComment = JSON.parse(classDefinition.schema.$comment);
+        
         openAPISpec.components.schemas[$classComment.term] = toOpenApi(
           classDefinition.schema,
         );
+        
+        const removeComments = function(obj) {
+          for(prop in obj) {
```

Comment:
```suggestion
          for(let prop in obj) {
```

Reply:
or `// eslint-disable-next-line` everywhere...

Reply:
yeah, playing with a few options here on the recursion locally to get it to behave just right